core/templates/dev/head/components/SolutionEditorDirective.js

```diff
@@ -79,7 +79,7 @@ oppia.directive('solutionEditor', [
                       stateInteractionIdService.savedMemento,
                       /* eslint-disable max-len */
                       explorationStatesService.getInteractionCustomizationArgsMemento(
-                        EditorStateService.getActiveStateName()),
+                        EditorStateService.getActiveStateName()), true,
```

Comment:
Move true to next line, to keep each arg starting on its own line.

Reply:
Done